rotkehlchen/tests/unit/decoders/test_main.py

```diff
@@ -57,3 +69,39 @@ def test_decoders_initialization(evm_transaction_decoder):
         'convex',
         'sushiswap-v2',
     }
+
+
+@pytest.mark.parametrize('ethereum_accounts', [['0xd8dA6BF26964aF9D7eEd9e03E53415D37aA96045']])  # noqa: E501
+def test_no_logs_and_zero_eth(
+        database,
+        ethereum_manager,
+        function_scope_messages_aggregator,
+):
+    """Data taken from
+    https://etherscan.io/tx/0x9a95424c48d36bb2f60fb7684a1068c08ec643c64144e7cdfbe5fb3fc820aa7f
+    """
+    tx_hash = deserialize_evm_tx_hash('0x9a95424c48d36bb2f60fb7684a1068c08ec643c64144e7cdfbe5fb3fc820aa7f')  # noqa: E501
+    # We don't need any events here, we just check that no errors occure during decoding
+    events = get_decoded_events_of_transaction(
+        ethereum_manager=ethereum_manager,
+        database=database,
+        msg_aggregator=function_scope_messages_aggregator,
+        tx_hash=tx_hash,
+    )
```

Comment:
Please don't query the transaction. Instead add the info manuall to the database and decode it like we do in tests for other decoders. This add an extra call to etherscan that can fail, is slow and would contribute to rate limits

Reply:
This is an okayish approach and used in some decoders tests (Lefteris also mentioned at some point that speed of writing decoders tests justifies longer tests running). But no problem, I can change it

Reply:
I don't believe is okey. You can try it and see how the time increases considerably making remote calls to etherscan vs hard coding them. Also in the places where someone used them we have
![image](https://user-images.githubusercontent.com/5068010/197225523-8e823d42-e598-44c4-bb0c-8f88365a410f.png)